npm/test-integration.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
```

Comment:
Since you're running Mocha directly, how will we get coverage stats from `istanbul`?